src/ui/views/preferencesdialog.cpp

```diff
@@ -25,6 +25,30 @@ PreferencesDialog::PreferencesDialog(GtkWindow* parent, const PreferencesDialogC
     adw_combo_row_set_model(ADW_COMBO_ROW(m_rowTheme), G_LIST_MODEL(gtk_string_list_new(new const char*[4]{ _("System"), _("Light"), _("Dark"), nullptr })));
     adw_preferences_group_add(ADW_PREFERENCES_GROUP(m_grpUserInterface), m_rowTheme);
     g_signal_connect(m_rowTheme, "notify::selected-item", G_CALLBACK((void (*)(GObject*, GParamSpec*, gpointer))[](GObject*, GParamSpec*, gpointer data) { reinterpret_cast<PreferencesDialog*>(data)->onThemeChanged(); }), this);
+    //Transaction Default Color Row
+    GdkRGBA transactionColor;
+    gdk_rgba_parse(&transactionColor, m_controller.getTransactionDefaultColor().c_str());
+    m_btnTransactionColor = gtk_color_button_new_with_rgba(&transactionColor);
```

Comment:
Loading the color should be moved to the `//Load Configuration` section of the constructor.

The buttons can be initialized with `gtk_color_button_new()` and the color can be then loaded with
```cpp
GdkRGBA transactionColor;
gdk_rgba_parse(&transactionColor, m_controller.getTransactionDefaultColor().c_str());
gtk_color_chooser_set_rgba(GTK_COLOR_CHOOSER(m_btnTransactionColor), &transactionColor);
```
in the appropriate section.